src-core/migrations/2025-06-09-150456_add_net_contribution_base_to_snapshots/up.sql

```diff
@@ -0,0 +1,13 @@
+-- Your SQL goes here
+
+ALTER TABLE holdings_snapshots
+ADD COLUMN net_contribution_base TEXT NOT NULL DEFAULT '0';
+
+
+-- Delete all quotes and snapshots to recalculate everything
+DELETE FROM quotes;
```

Comment:
@afadil this PR includes a migration that deletes all quotes, including the MANUAL sources that are not recoverable by updating the portfolio using a market service.

This should probably be 
```sql
DELETE FROM quotes where data_source <> 'MANUAL';
```

Reply:
Alternatively if it's just the filled-in quotes that were introduced recently, the least impactful change would be
```sql
DELETE FROM quotes where id like '%filled';
```